apollo-federation/src/query_plan/display.rs

```diff
@@ -364,6 +357,69 @@ fn write_selections(
     state.write("}")
 }
 
+fn write_requires_selections(
+    state: &mut State<'_, '_>,
+    mut selections: &[requires_selection::Selection],
+) -> fmt::Result {
+    if let Some(requires_selection::Selection::Field(field)) = selections.first() {
```

Comment:
Requires selections won't have `_entities`. So, this if statement won't be necessary.